index.html

```diff
@@ -105,6 +98,28 @@ <h1 class="about-title">About Myself</h1>
                 </ul>
             </div>
         </section>
+        <section id="contact-section" class="contact-section">
+            <div class="contact">
+                <div class="contact-info">
+                    <div>
+                        <h2>Contact me</h2>
+                        <p>if you have an application you are interested in developing, a feature you that you need built or a project that needs coding. I'd love to help with it!</p>
+                    </div>
+                    <form class="contact-form" action="https://formspree.io/f/mzbqjzeq" method="post">
+                        <input class="form-style" type="text" name="name" id="full-name" placeholder="Please Enter your name">
+                        <span id="fullNameErr"></span>
+                        <input class="form-style" name="email" id="email" placeholder="Please Enter your email address" required>
+                        <span id="emailErr"></span>
+                        <textarea class="form-style" name="message" id="message" cols="30" rows="10" placeholder="Write your message here" ></textarea>

```

Comment:
- [ ] Once again as part of the project requirements, please ensure that the message input field has a `maxlength` attribute set to 500. This helps to prevent users from entering excessively long messages as well which may cause issues in the application. Thank you for your attention to this matter. Also please make sure to add the `required` attribute for this input as well.